canopen/node/local.py

```
@@ -127,3 +138,32 @@ def _find_object(self, index, subindex):
                 raise SdoAbortedError(0x06090011)
             obj = obj[subindex]
         return obj
+
+    def _nmt_state_changed(self, old_state, new_state):
+        if new_state == "OPERATIONAL":
+            for pdo in self.tpdo.map.values():
+                if pdo.enabled:
+                    try:
+                        pdo.start()
+                        logger.info("Successfully started %s", pdo.name)
+                    except ValueError:
+                        logger.warning("Failed to start %s due to missing period", pdo.name)
+                    except Exception:
+                        logger.exception("Unknown error starting %s", pdo.name)
+                else:
+                    logger.info("%s not enabled", pdo.name)
+        else:
+            self.tpdo.stop()
+
+    def _tpdo_configuration_write(self, tpdoNum):
+        pdo = self.tpdo.map[tpdoNum]
+
+        # Only allowed to edit pdo configuration in pre-op or operational
+        if self.nmt.state not in ("PRE-OPERATIONAL", "OPERATIONAL"):
+            logger.warning("Tried to configure %s when not in pre-op or operational", pdo.name)
+            return
+        
+        try:
+            pdo.read(from_od=True)
+        except:
+            pass
```

Why would we want to silently swallow any and all exceptions here?